packages/user/src/model/users/controller.ts

```diff
@@ -48,6 +52,26 @@ const plugin = async (
     }
   );
 
+  fastify.get(
+    ROUTE_CURRENT_USER,
+    {
+      preHandler: fastify.verifySession(),
+    },
+    async (request: SessionRequest, reply: FastifyReply) => {
+      const service = new Service(request.config, request.slonik);
+
+      const session = await Session.getSession(request, wrapResponse(reply), {
```

Comment:
Since, you have used pre-handler the request will have session variable. so you can just do session = request.session to get session.